dlc-manager/src/manager.rs

```diff
@@ -344,7 +356,11 @@ where
         offered_message: &OfferDlc,
         counter_party: PublicKey,
     ) -> Result<(), Error> {
-        offered_message.validate(&self.secp, REFUND_DELAY, REFUND_DELAY * 2)?;
+        offered_message.validate(
+            &self.secp,
+            self.options.refund_delay,
+            self.options.refund_delay * 2,
```

Comment:
Sure, i'll make that change. More flexibility is always good.

@Tibo-lg What is the intention of having a minRefundDelay and a max? What does it do to have those bounds, rather than just 1 value and making the refund after that time?

Reply:
I created a type called RefundDelayWindow. Tell me what you think.